content/docs/command-reference/plots/modify.md

```diff
@@ -123,6 +123,29 @@ plots:
       title: Accuracy
 ```
 
+## Example: Make a regular output into a plot
+
+We can use `dvc plots modify` to make an <abbr>output</abbr> or metrics file of
+a stage into a plot. Lets take an example stage `evaluate` which has an
+<abbr>output</abbr> `logs.csv`:
+
+```yaml
+evaluate:
+  cmd: python src/evaluate.py
+    deps:
+    - src/evaluate.py
+    outs:
+    - logs.csv
+    plots:
+    - scores.json
+```
+
+To change the output file `logs.csv` into a plot run:
+
+```dvc
+$ dvc plots modify logs.csv
+```
```

Comment:
Maybe say what the operation is going to be and then use a ` ```git ` (instead of YAML) with `-` and `+` rows showing the `out` being moved to `plots`.

Reply:
> `-` and `+` rows showing the file being moved from outs to plots

Updated the example with this.

Reply:
Thanks, looks great in https://dvc-landing-plotscmd-kdw3xfowb.herokuapp.com/doc/command-reference/plots/modify#example-convert-any-output-into-a-plot!